<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSymbolLegandre</class>
 <widget class="QDialog" name="DialogSymbolLegandre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Диапазон генерации</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>От:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineMinA"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>До:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineMaxA"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>От:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineMinP"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>До:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineMaxP"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>p</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Модификаторы генерации:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="btnDefault">
       <property name="text">
        <string>По умолчанию</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinDefault"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="btnPrimes">
       <property name="text">
        <string>Раскладывается в произведение простых</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinPrimes"/>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="btn1">
       <property name="text">
        <string>В качестве а = -1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spin1"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="btn2">
       <property name="text">
        <string>В качестве а = 2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spin2"/>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="btnNotEvenPrimes">
       <property name="text">
        <string>a и p взаимно просты и нечетны</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinNotEvenPrimes"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSymbolLegandre</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSymbolLegandre</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
